Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -1023,6 +1023,24 @@ Example::
 		node = ${q(node).parent().get(0)}
 	}
 
+
+.. _Neos_Neos__NodeLink:
+
+Neos.Neos:NodeLink
+-----------------
+
+Renders an anchor tag pointing to the node given via the argument. Based on `Neos_Neos__NodeUri`.
+
+:\*: All :ref:`Neos_Neos__NodeUri` properties
+:attributes: (:ref:`Neos_Fusion__Attributes`) Link tag attributes
+:content: (string) The label of the link.
```

Comment:
I'm missing here the description of the default value

Reply:
Correct, is adjusted